modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -135,36 +135,39 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
-  name: 98-var-log-partition
+  name: 98-var-partition
 spec:
   config:
     ignition:
       version: 3.1.0
     storage:
       disks:
-      - device: /dev/nvme0n1
+      - device: /dev/<device_name> <1>
         wipeTable: false
```

Comment:
`wipeTable: false` is the default, so let's remove it.

Reply:
Resolved